src/components/CaptureDetailDialog.js

```diff
@@ -288,7 +287,7 @@ function CaptureDetailDialog(props) {
               label={verificationStates.AWAITING}
               className={classes.awaitingChip}
             />
-          ) : capture.status === 'approved' ? (
+          ) : capture.status === 'active' || capture.status === 'approved' ? (
```

Comment:
What's this `active` status?
Also, I think you know what I'm going to say about using string literals for comparison 😜 - best to define them once as a `const` object and reference that.